### DISCUSSION TITLE
Curious about the versioning of the sdk

### DISCUSSION THREAD

**Discussion by shailen-naidoo** [nikspyratos/investec-sdk-php#13] [General]
Hey Nik, I had a look through your SDK and it is quite cool. I am not the biggest fan of PHP but your documentation reads really well and I would say that the API is really nice. Something did catch my eye and that is the versioning of the SDK, I was curious as to how many releases did you go through before you published the v1.0.0 version?

**Comment by nikspyratos** [comment]
Heyo.

Didn't do any other releases, just worked on it until the Private Banking API functionality was fully implemented with both auth options. With tests working, I declared that v1.

The majority of users of this are either going to be developers accessing their own account data, or third parties accessing someone else's account data, so that's where I put most of my focus.

Next release is going to be using the latest versions of dependencies and also publishing the other API sections - though labelled as experimental. I don't use most of them and didn't get tests working for them either last time I tried, but that code might still be useful so I feel it should still be included.

**Reply by shailen-naidoo** [reply]
Thanks for your response! Makes sense, I was merely curious!
